src/Library/demos/Color Dialog/main.js

```diff
@@ -0,0 +1,42 @@
+import Gtk from "gi://Gtk";
+import Gdk from "gi://Gdk";
+import Gio from "gi://Gio";
+
+Gio._promisify(Gtk.ColorDialog.prototype, "choose_rgba", "choose_rgba_finish");
+
+const color_dialog_button = workbench.builder.get_object("color_dialog_button");
+const custom_button = workbench.builder.get_object("custom_button");
+
+const color = new Gdk.RGBA();
+color.parse("white");
+
+const dialog_standard = new Gtk.ColorDialog({
+  title: "Select a color",
+  modal: true,
+  with_alpha: true,
+});
+
+color_dialog_button.dialog = dialog_standard;
+color_dialog_button.rgba = color;
+
+color_dialog_button.connect("notify::rgba", () => {
+  console.log(
```

Comment:
If this is only for debugging use (i.e. you're not printed to `stdout` for another process to read), use `console.debug()` instead.

Reply:
Okay, I wasn't aware of that. I only used ``console.log`` because that's what all the demos have been using even though they are not actually being read by another process. As a demo, I think it's okay to be more verbose? Since the main purpose of that snippet is to show how to retrieve the color from a dialog.

Reply:
Yup, lots of output is good. The different between `console.log()` and `console.debug()` is that debug is usually filtered out for users, but easy to turn on for developers.

No big deal though, you're right this whole application is for developers :)